src/Converter/ShipmentLineItemsConverter.php

```diff
@@ -50,14 +57,25 @@ private function convertUnitRefundToLineItem(ShipmentRefund $shipmentRefund): Li
         Assert::isInstanceOf($shipment, AdjustableInterface::class);
         Assert::lessThanEq($shipmentRefund->total(), $shipment->getAdjustmentsTotal());
 
+        $grossValue = $shipmentRefund->total();
+
+        $taxAdjustment = $shipment->getAdjustments(AdjustmentInterface::TAX_ADJUSTMENT)->first();
+
+        $taxAdjustmentAmount = $this->taxRateAmountProvider->provide($taxAdjustment);
+        $taxRate = $taxAdjustmentAmount * 100 . '%';
+        $taxAmount = (int) ($grossValue * $taxAdjustmentAmount);
+
+        $netValue = $grossValue - $taxAmount;
+
```

Comment:
I would divide it like this where there are 2 groups: 1st values are getted, and 2nd are calculated 😄 wdyt?
```suggestion
        $grossValue = $shipmentRefund->total();
        $taxAdjustment = $shipment->getAdjustments(AdjustmentInterface::TAX_ADJUSTMENT)->first();
        $taxAdjustmentAmount = $this->taxRateAmountProvider->provide($taxAdjustment);
        
        $taxRate = $taxAdjustmentAmount * 100 . '%';
        $taxAmount = (int) ($grossValue * $taxAdjustmentAmount);
        $netValue = $grossValue - $taxAmount;
```